src/sdl2/render.rs

```diff
@@ -689,8 +691,10 @@ impl<'a> Renderer<'a> {
 
     /// Draws a line on the current rendering target.
     /// Errors if drawing fails for any reason (e.g. driver failure)
-    pub fn draw_line(&mut self, start: Point, end: Point)
+    pub fn draw_line<P: Into<Point>>(&mut self, start: P, end: P)
```

Comment:
That's a very small detail but you should make two `P: Into<Point>` here. If you only use one, start will have to be the same type as end, whereas with P1 and P2 they won't have to be the same type.